<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PythonScriptingEditor</class>
 <widget class="QDialog" name="PythonScriptingEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editor de scripts Python</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="editorContainer" native="true">
      <layout class="QVBoxLayout" name="editorLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPlainTextEdit" name="editor">
         <property name="toolTip">
          <string>Escriba su código Python aquí. Use Etapas.get_all() para obtener todas las etapas.</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="editorButtonLayout">
         <item>
          <widget class="QToolButton" name="btExecute">
           <property name="text">
            <string>Ejecutar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btDocumentation">
           <property name="text">
            <string>Documentación</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btSaveScript">
           <property name="text">
            <string>Guardar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btLoadScript">
           <property name="text">
            <string>Cargar</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="btClearEditor">
           <property name="text">
            <string>Borrar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="findReplaceGroup">
         <property name="title">
          <string>Buscar/Reemplazar</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="findLayout">
            <item>
             <widget class="QLabel" name="lblFind">
              <property name="text">
               <string>Buscar:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="edFind"/>
            </item>
            <item>
             <widget class="QToolButton" name="btFind">
              <property name="text">
               <string>Buscar</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="replaceLayout">
            <item>
             <widget class="QLabel" name="lblReplace">
              <property name="text">
               <string>Reemplazar:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="edReplace"/>
            </item>
            <item>
             <widget class="QToolButton" name="btReplace">
              <property name="text">
               <string>Reemplazar</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="btReplaceAll">
              <property name="text">
               <string>Reemplazar Todo</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="optionsLayout">
            <item>
             <widget class="QCheckBox" name="cbCaseSensitive">
              <property name="text">
               <string>Coincidir mayúsculas/minúsculas</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cbWholeWords">
              <property name="text">
               <string>Palabras completas</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="consoleContainer" native="true">
      <layout class="QVBoxLayout" name="consoleLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Resultados de la ejecución</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="result">
         <property name="styleSheet">
          <string notr="true">background: #333;color:#fdfdfd;</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="consoleButtonLayout">
         <item>
          <widget class="QToolButton" name="btCopyConsole">
           <property name="text">
            <string>Copiar</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="btClearConsole">
           <property name="text">
            <string>Limpiar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
